package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "pytest-html",
+  "version": "3.0.0",
```

Comment:
My experience with nodejs is low but I would really try to avoid having an extra new place to update the versions. The code should not keep the version, this should be takes from git (scm) tags, like setuptools-scm does.

Reply:
@ssbarnea I can try removing it and seeing if the install and test target still run fine. I don't see a need to have one since we're not publishing to npm. 

I'll try when I'm free this afternoon.